grails-app/views/email/_approvalsStatusChanged.gsp

```diff
@@ -0,0 +1,60 @@
+<%@ page contentType="text/html"%>
+<style>
+    label {
+        font-weight: bold;
+    }
+</style>
+<div>
+    <div class="header">
+        <g:render template="/email/header"/>
+    </div>
+    <div>
+        <g:message code="email.statusChange.message" args="[format.metadata(obj:requisition.status), g.createLink(uri: '/', absolute: true)]" />
+    </div>
+    <div>
+        <g:message code="email.withComment.message" />
+        <g:set var="comment" value="${requisition.recentComment}" />
```

Comment:
i assume we can't run into a `NullPointerException` here, because it would not be rendered if we don't have requisitoin?